swarm/catalog/swarm/swarm.bom

```diff
@@ -6,15 +6,6 @@ brooklyn.catalog:
   - mvn:io.brooklyn.etcd/brooklyn-etcd/2.7.0-SNAPSHOT # BROOKLYN_ETCD_VERSION
 
   items:
-  - id: docker-swarm-template
```

Comment:
I think we need to leave this in case someone's persisted state references it.

However, we should change it to `itemType: entity` - that will look a bit weird (given the entity name), but it's not for the swarm bundle to say what should go in "quick launch". That should be up to the person who is configuring brooklyn to decide which of their bundle's entities should be "quick launch" items.